recipe/meta.yaml

```diff
@@ -19,13 +24,15 @@ requirements:
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - pybind11                               # [build_platform != target_platform]
-    - {{ compiler('fortran') }}   # [unix]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    # pythran code needs clang on windows
+    - {{ compiler('clang') }}                # [win]
```

Comment:
Don't use this. This is not using `clang-cl`, but `clang`. Go back to just `- clang` and set env variables `CC` and `CXX` to `clang-cl`

Reply:
@isuruf Should we make a `compiler('clang-cl')` activation once things work here? I found it quite useful to have these reusable activation scripts.

Reply:
A couple commits ago, I just had `- clang`, but then `clang-cl` was not on the `PATH`. I assume I'll also need add something to the `PATH`?

Reply:
Nope. It should be on the PATH.

Reply:
https://github.com/conda-forge/clang-win-activation-feedstock/blob/724a3900c88ef1748a9c1d90d0c2927db99c3ef5/recipe/activate-clang_win-64.bat#L1 isn't adding anything to the `PATH`, so it must be that you are missing a dependency that is specified here: https://github.com/conda-forge/clang-win-activation-feedstock/blob/724a3900c88ef1748a9c1d90d0c2927db99c3ef5/recipe/meta.yaml#L41

Reply:
As I said earlier, none of those dependencies are needed. Just `clang`

Reply:
Just `- clang` does not add `clang-cl` to the PATH (the presence-check I added in `bld.bat` is failing):
```
(%BUILD_PREFIX%) %SRC_DIR%>clang-cl.exe --version 

(%BUILD_PREFIX%) %SRC_DIR%>if -1073741515 NEQ 0 exit 1 
```